### PR TITLE
feat(ux): better error messages

### PR DESCRIPTION
**Pull request by hacdias**
This adds the possibility of adding custom error messages. I also updated some places to show specific error messages instead of general ones.

### CODE REVIEW

File: assets/locales/en.json [hunk 1]
```diff
@@ -141,5 +141,13 @@
   "runGarbageCollectorDone": {
     "title": "Garbage collector",
     "message": "The garbage collector ran successfully."
+  },
+  "moveRepositoryFailed": {
+    "title": "Move repository",
+    "message": "Could not move the repository from { currDir } to { newDir }."
```
Comment (jessicaschilling) [inline assets/locales/en.json:147]: Put double quotes around the values of currDir and newDir? (Sorry to not suggest code itself, am on my phone!)

File: assets/locales/en.json [hunk 2]
```diff
@@ -141,5 +141,13 @@
...
+  },
+  "cantAddIpfsToPath": {
+    "title": "IPFS on PATH",
```
Comment (jessicaschilling) [inline assets/locales/en.json:150]: “IPFS on path”? Not sure why we’d uppercase “path” here.

File: assets/locales/en.json [hunk 3]
```diff
@@ -141,5 +141,13 @@
...
+    "message": "Could not add IPFS to the PATH."
```
Comment (jessicaschilling) [inline assets/locales/en.json:151]: “... to the path”? Not sure why we’d uppercase “path” here.

File: assets/locales/en.json (hunk 2 above)
Comment (hacdias) [reply assets/locales/en.json:150]: We're referring to the environment variable PATH so we usually put that in uppercase 😃

Comment (jessicaschilling) [reply assets/locales/en.json:150]: Ah right, never mind!

**Pull request merged**